code/modules/client/client_procs.dm

```diff
@@ -1048,19 +1048,22 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 					var/ooc = tgui_say_create_open_command(OOC_CHANNEL)
 					winset(src, "default-[REF(key)]", "parent=default;name=[key];command=[ooc]")
 
-/client/proc/change_view(new_size)
-	if (isnull(new_size))
-		CRASH("change_view called without argument.")
-
-	view = new_size
-	SEND_SIGNAL(src, COMSIG_VIEW_SET, new_size)
-	mob.hud_used.screentip_text.update_view()
-	apply_clickcatcher()
-	mob.reload_fullscreen()
-	if (isliving(mob))
-		var/mob/living/M = mob
-		M.update_damage_hud()
-	attempt_auto_fit_viewport()
+/client/proc/change_view(new_size, forced)
+	if((!prefs?.read_preference(/datum/preference/toggle/widescreen)))
+		if (isnull(new_size))
+			CRASH("change_view called without argument.")
+		view = new_size
+		apply_clickcatcher()
+		mob?.reload_fullscreen()
+		if (isliving(mob))
+			var/mob/living/M = mob
+			M.update_damage_hud()
+		attempt_auto_fit_viewport()
+	else
+		apply_clickcatcher()
+		mob?.reload_fullscreen()
+		if(forced)
+			view = new_size
```

Comment:
why drop the stuff you do here? also I still think an early return would be better